external/test/crd-clusterwidenetworkpolicy.yaml

```diff
@@ -0,0 +1,192 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Use `helm template charts/postgreslet --show-only templates/configmap-sidecars.yaml` to generate the file

Reply:
For cm-sidecar.yaml

Reply:
You can also add a name:
`helm template envtest charts/postgreslet --show-only templates/configmap-sidecars.yaml`